crates/dbsp/src/operator/recursive.rs

```diff
@@ -76,7 +76,7 @@ where
     }
 
     fn export(self) -> Self::Export {
-        self.integrate_trace().export()
+        self.integrate_trace_in_memory().export()
```

Comment:
should this be integrate_trace(Layer.MEMORY)?

Reply:
I don't understand this comment. Can you expand on it for me?

Reply:
Is there an integrate_trace_on_disk()?
Then I am suggesting having one function with an argument instead of two functions.

Reply:
I don't think that's possible, because the two functions return different types.

```rust
    pub fn integrate_trace_in_memory(&self) -> Stream<C, Spine<B>>;
    pub fn integrate_trace(&self) -> Stream<C, Spine<<B as SpillableBatch>::Spilled>>;
```